spec/openapi.yaml

```diff
@@ -314,12 +348,83 @@ paths:
                 format: binary
         401:
           $ref: "#/components/responses/Unauthorized"
+        403:
+          $ref: "#/components/responses/Forbidden"
+        404:
+          $ref: "#/components/responses/NotFound"
+        408:
+          $ref: "#/components/responses/RequestTimeout"
+        409:
+          $ref: "#/components/responses/Conflict"
+        503:
+          $ref: "#/components/responses/ServiceUnavailable"
+
+  /api/v1/cameras/{id}/config:
+    patch:
+      summary: Set new settings to a working camera
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CameraConfigSet"
+      responses:
+        200:
+          description: OK
+        401:
+          $ref: "#/components/responses/Unauthorized"
+        403:
+          $ref: "#/components/responses/Forbidden"
+        404:
+          $ref: "#/components/responses/NotFound"
+        503:
+          $ref: "#/components/responses/ServiceUnavailable"
+    delete:
+      summary: Reset settings of a camera
+      responses:
+        200:
+          description: OK
+        401:
+          $ref: "#/components/responses/Unauthorized"
+        403:
+          $ref: "#/components/responses/Forbidden"
+        404:
+          $ref: "#/components/responses/NotFound"
+        503:
+          $ref: "#/components/responses/ServiceUnavailable"
+
+  /api/v1/cameras/{id}/connection:
```

Comment:
[question]
Will there a parameter somewhere that camera is connected to the cloud?